app/page.tsx

```diff
@@ -18,6 +18,6 @@ export default async function Index() {
             </div>
         </div>
     ) : (
-        <h1 className="text-3xl font-bold">No Upcoming Competetions</h1>
+        <h1 className="text-2xl font-bold">No Upcoming Competetions</h1>
```

Comment:
- [ ] keep the text's property similar to above (upcoming competitions) 
- [ ] change competetions to competitions

Reply:
fixed